client/src/locales/ja/core.js

```diff
@@ -23,7 +24,7 @@ export default {
       administrator: '管理者',
       all: 'すべて',
       allChangesWillBeAutomaticallySavedAfterConnectionRestored:
-        '全ての変更は接続回復後自動的に保存されます',
+        '全ての変更は接続回復後自動的に保存されます。',
```

Comment:
For example here

Reply:
I will check it :)

Reply:
1. Attempts to express the intent of the original text to the best of its ability.
2. The punctuation is a faithful reflection of the original text.
  (If a sentence ends without a full stop, it is because it is faithful to the original text.)
3. Sentences ending in English verbs end in Korean (or Japanese) verbs, and sentences ending in English nouns end in Korean (or Japanese) nouns.
4. Honorifics in Korean and Japanese have subtle differences from those in English, and I have endeavoured to translate them accordingly.